server/routes/flow_framework_routes_service.ts

```diff
@@ -58,6 +60,19 @@ export function registerFlowFrameworkRoutes(
     flowFrameworkRoutesService.getWorkflow
   );
 
+  router.get(
+    {
+      path: `${BASE_NODE_API_PATH}/{dataSourceId}/workflow/{workflow_id}`,
```

Comment:
nit: common pattern has the vars in snake case - could you update `dataSourceId` -> `data_source_id`?